packages/components/src/modal/style.scss

```diff
@@ -20,7 +20,7 @@
 	width: 100%;
 	background: $white;
 	box-shadow: $shadow-modal;
-	border-radius: $radius-block-ui;
+	border-radius: $grid-unit;
```

Comment:
Should probably be `$grid-unit-10` for accuracy, right? This will work, but I see the `$grid-unit` mostly as the base grid lacking a multiplication factor. I know that's nitpicky!